src/hmem_cuda.c

```diff
@@ -760,6 +771,20 @@ bool cuda_is_gdrcopy_enabled(void)
 	return hmem_cuda_use_gdrcopy;
 }
 
+#if CUDART_VERSION >= 11030
```

Comment:
I think you need to move this macro to the code that assigns `cuda_ops.cudaDeviceFlushGPUDirectRDMAWrites`. See my other 2 comments for dlopen/non-dlopen cases.

Reply:
agreed.